### PR TITLE
web platform august post

### PR DESCRIPTION
**Pull request by rachelandrew**

### CODE REVIEW

File: src/site/content/en/blog/web-platform-08-2022/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+Firefox 104 adds support for the methods [Array.prototype.findLast()](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/findLast), 
+[Array.prototype.findLastIndex()](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/findLastIndex), 
+[TypedArray.prototype.findLast()](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/findLast), and [TypedArray.prototype.findLastIndex()](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/findLastIndex). These are used to find the value and index (respectively) of the last element in an Array or TypedArray that matches a supplied test function.
+
+{% BrowserCompat 'javascript.builtins.Array.findLast' %}
+
+### Sanitizer API
+
+ALso in Chrome 105 is the [Sanitizer API](https://developer.chrome.com/blog/new-in-chrome-105/#sanitizer-api). This API builds sanitization into the platform to help remove cross-site scriptig vulnerabilities.
```
Comment (andreban) [inline src/site/content/en/blog/web-platform-08-2022/index.md:99]:
```suggestion
Also in Chrome 105 is the [Sanitizer API](https://developer.chrome.com/blog/new-in-chrome-105/#sanitizer-api). This API builds sanitization into the platform to help remove cross-site scriptig vulnerabilities. 
```

File: src/site/content/en/blog/web-platform-08-2022/index.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+}
+```
+
+{% BrowserCompat 'css.at-rules.media.range_syntax' %}
+
+### Container queries
+
+Chrome 105 is an exciting release bringing the long-awaited feature of container queries to the web platform. While media queries give you a way to query against the size of the viewport, container queries provide a method of querying against the size of a container.
+
+{% BrowserCompat 'css.at-rules.container' %}
```
Comment (andreban) [inline src/site/content/en/blog/web-platform-08-2022/index.md:54]: I'm actually not seeing this on the preview, for some reason.

Comment (andreban) [reply src/site/content/en/blog/web-platform-08-2022/index.md:54]: Per conversation, it's being added via another PR. So, resolving it.

File: src/site/content/en/blog/web-platform-08-2022/index.md (hunk 1 above)
Comment (matthiasrohmer) [inline src/site/content/en/blog/web-platform-08-2022/index.md:99]:
```suggestion
Also in Chrome 105 is the [Sanitizer API](https://developer.chrome.com/blog/new-in-chrome-105/#sanitizer-api). This API builds sanitization into the platform to help remove cross-site scripting vulnerabilities. 
```

**Review by matthiasrohmer — APPROVED** [review]

**Pull request merged**